test/transcoding/fp_contract_reassoc_fast_mode.ll

```diff
@@ -0,0 +1,57 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -spirv-text %t.bc -o - | FileCheck %s --check-prefix=CHECK-SPIRV-OFF
+; RUN: llvm-spirv --spirv-ext=+SPV_INTEL_fp_fast_math_mode -spirv-text %t.bc -o - | FileCheck %s --check-prefix=CHECK-SPIRV-ON
+; RUN: llvm-spirv  --spirv-ext=+SPV_INTEL_fp_fast_math_mode %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
```

Comment:
That is why the test is failing: spirv-val doesn't know about the added capability here (and PR https://github.com/KhronosGroup/SPIRV-Headers/pull/176, where the extension is added to SPIR-V Headers isn't yet merged). I suggest to remove it for now.